packages/govern-create/test/base.ts

```diff
@@ -0,0 +1,63 @@
+import { deployments, ethers, network, waffle } from 'hardhat'
+import { expect } from 'chai'
+import { GovernBaseFactory, GovernRegistry } from '../typechain'
+
+const EVENTS = {
+  REGISTERED: 'Registered',
+  SET_METADATA: 'SetMetadata',
+}
+
+describe('Govern Base Factory', function () {
+  beforeEach(async () => {
+    await deployments.fixture()
+  })
+
+  const deployDAO = async (
+    useProxies: boolean,
+    gasTarget: number,
+    deployToken = false
+  ) => {
+    const registryDeployment = await deployments.get('GovernRegistry')
+    const registryContract = (await ethers.getContractAt(
+      'GovernRegistry',
+      registryDeployment.address
+    )) as GovernRegistry
+
+    const baseFactoryDeployment = await deployments.get('GovernBaseFactory')
+    const baseFactoryContract = (await ethers.getContractAt(
+      'GovernBaseFactory',
+      baseFactoryDeployment.address
+    )) as GovernBaseFactory
+
+    const tx = baseFactoryContract.newGovernWithoutConfig(
+      'eagle',
+      `0x${(deployToken ? '00' : '11').repeat(20)}`, // NOTE: zero addr deploys a token
+      'Eaglet Token',
+      'EAG',
+      useProxies
+    )
+
+    await expect(tx).to.emit(registryContract, EVENTS.REGISTERED)
+    await expect(tx).to.emit(registryContract, EVENTS.SET_METADATA)
+
+    const { hash } = await tx
+    const { gasUsed } = await waffle.provider.getTransactionReceipt(hash)
+
+    expect(gasUsed).to.be.lte(gasTarget)
```

Comment:
Wouldn't it be lower than or equals?

Reply:
As I understood this is what `lte` is doing: https://www.chaijs.com/api/bdd/#method_most

Reply:
Brainfart the second.